changes/test_tasks.py

```diff
@@ -311,6 +362,121 @@ def test_no_message_found(self, mock_create_outbound):
         )
         self.assertFalse(mock_create_outbound.called)
 
+    @mock.patch("changes.tasks.utils.ms_client.create_outbound")
+    @mock.patch("changes.tasks.is_client.update_identity")
+    @responses.activate
+    def test_timeout_error(self, mock_update_identiity, mock_create_outbound):
+        """
+        The task should create a Change according to the details received from the
+        message sender
```

Comment:
This docstring is incorrect. Should probably say something about sending an outbound message

Reply:
thanks for picking this up